pages/tools.jsx

```diff
@@ -1,7 +1,9 @@
 var React = require('react');
-var Illustration = require('../components/illustration.jsx');
 var Router = require('react-router');
 var Link = Router.Link;
+var ga = require('react-ga');
+var OutboundLink = ga.OutboundLink;
```

Comment:
since we don't use `ga` anywhere else, we can probably just say `var OutboundLink = require('react-ga').OutboundLink;`